### ISSUE TITLE
NPE when connecting with light wallet to IRI

### ISSUE THREAD

**Issue by georgmittendorfer** [iotaledger/iri#513]
```
01/28 21:03:50.693 [XNIO-1 task-18] ERROR com.iota.iri.service.API - API Exception: 
java.lang.NullPointerException: null
        at com.iota.iri.service.API.exhaustiveSearchWithinIndex(API.java:676) [iri-1.4.1.7.jar:na]
        at com.iota.iri.service.API.getNewInclusionStateStatement(API.java:661) [iri-1.4.1.7.jar:na]
        at com.iota.iri.service.API.process(API.java:243) [iri-1.4.1.7.jar:na]
        at com.iota.iri.service.API.processRequest(API.java:171) [iri-1.4.1.7.jar:na]
        at com.iota.iri.service.API.access$000(API.java:68) [iri-1.4.1.7.jar:na]
        at com.iota.iri.service.API$1.handleRequest(API.java:152) [iri-1.4.1.7.jar:na]
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:208) [iri-1.4.1.7.jar:na]
        at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:809) [iri-1.4.1.7.jar:na]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_162]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_162]
        at java.lang.Thread.run(Thread.java:748) [na:1.8.0_162]
```
This happend at the node when trying to connect to IRI with light wallet. I entered the seed and pressed connect in the wallet. It worked before. Other API calls were working. Maybe it has something to do with the current snapshot: I did the snapshot validation in the ixi folder and restarted IRI before having this error.

Whatever the reason there should be some error message instead of the NPE...

**Comment by rajivshah3** [comment]
🤔  I wonder if this is related to #463

**Comment by alon-e** [comment]
should be fixed by #506 , please reopen if not.
